utils/fields.py

```diff
@@ -83,9 +89,22 @@ def generate_fake_data(self, faker, **kwargs):
         :param kwargs:
         :return: list
         """
-        num_items = randint(self.min_items, self.max_items)
         fake_data = []
 
+        if (overrides := kwargs.get("overrides")) and self.name in overrides:
+            override_array = overrides[self.name]
+
+            if isinstance(override_array, list):
+                for item in override_array:
+                    kwargs["overrides"] = item
+                    fake_data.append(self.items_field.generate_fake_data(faker, **kwargs))
+
+                return fake_data
+
+            kwargs["overrides"] = override_array
+
+        num_items = randint(self.min_items, self.max_items)
```

Comment:
how will this handle payload attributes, like the `MTOAgents`, that have a `maxItems` attribute?

Reply:
The parser will grab the `minItems` and `maxItems` values from the YAML file and set them as `self.min_items` and `self.max_items` here. That way the `num_items` will always be a random number between those edges. So for MTO Agents, it would always either be 1 or 2 (since we test with all fields as required)